python/ray/tune/schedulers/hyperband.py

```diff
@@ -274,9 +287,11 @@ def _process_bracket(
                         "If you encounter this, please file an issue on the Ray Github."
                     )
                     if t.status == Trial.PAUSED:
+                        logger.debug(f"Unpausing trial {str(t)}")
                         self._unpause_trial(trial_runner, t)
                         trial_runner._set_trial_status(t, Trial.PENDING)
                     elif t.status == Trial.RUNNING:
+                        logger.debug(f"Continuing current trial {str(t)}")
                         action = TrialScheduler.CONTINUE
```

Comment:
Can we also add some comments explaining the `RUNNING` == current trial?

Reply:
Added to the main docstring and in-line comments!